x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -15,6 +15,10 @@ minutes. This token automatically applies the security settings from your {es}
 cluster, authenticates to {es} with the built-in `kibana` service account, and writes the
 security configuration to `kibana.yml`.
 
+NOTE: There are <<stack-skip-auto-configuration,some cases>> where security
+can't be configured automatically because an existing environment or existing
+settings are detected when starting {es}.
```

Comment:
```suggest
will not be configured automatically because it appears that the node is already part of a cluster, Security (or just TLS) is already configured or explicitly disabled.
```  

It's only a suggestion feel free to adapt it, as you see fit.

Reply:
I'll change this to read:

>...because the node startup process detects that the node is already part of a cluster, or that security is already configured or explicitly disabled.